test/acceptance/resources/backend.operator.manifest.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: backends.stable.example.com
+spec:
+  group: stable.example.com
+  versions:
+    - name: v1
+      served: true
+      storage: true
+  scope: Namespaced
+  names:
+    plural: backends
+    singular: backend
+    kind: Backend
+    shortNames:
+      - bk
+---
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
I think CSV could be abridged and then CRD and CSV could be added to feature file itself. https://github.com/redhat-developer/service-binding-operator/pull/620#discussion_r478566551

Reply:
CSV has been abridged by removing roles, permissions etc. Can you be a bit more specific as to what potentially can be removed from here? 
I am omitting to add it to the feature file and adding to test/acceptance/resources, so that in the future other feature files can directly use the yaml files instead of mentioning the crd and csv yet again.

Reply:
Cool, we can keep it.